src/fides/api/ctl/migrations/versions/8173a96aad1a_populate_default_notice_templates.py

```diff
@@ -0,0 +1,33 @@
+"""populate default notice templates
+
+Revision ID: 8173a96aad1a
+Revises: d8a50048dfed
+Create Date: 2023-04-20 21:23:08.321999
+
+"""
+from fides.api.ops.util.consent_util import load_default_notices
+from fides.core.config import CONFIG
+
+# revision identifiers, used by Alembic.
+from fides.lib.db.session import get_db_session
+
+revision = "8173a96aad1a"
+down_revision = "d8a50048dfed"
+branch_labels = None
+depends_on = None
+
+DEFAULT_PRIVACY_NOTICES_PATH = (
+    "/fides/data/privacy_notices/privacy_notice_templates.yml"
+)
+
+
+def upgrade():
+    """Add default privacy notices to the database, and then create
+    PrivacyNotices and PrivacyNoticeHistories from there"""
+    sessionlocal = get_db_session(CONFIG)
+    with sessionlocal() as session:
+        load_default_notices(session, DEFAULT_PRIVACY_NOTICES_PATH)
```

Comment:
I generally think it is not a great practice to use SQLAlchemy models within a migration as I"m doing here, as the models themselves can change, making the data migration at this point in time invalid.  But I think this would be caught in development if this happened in the future, and I think the benefits are worth all the extra validation we get here.  

Alternatively, this could be something that runs on startup, perhaps if there are no Privacy Notice Templates in the database at all?

Reply:
Ah, silly, I can't create this in a data migration, it has to go in on startup anyway, which is fine, this works better.  Wrapped in a try/except, if it fails, we just continue to bring everything up.